src/components/Editor/Editor.tsx

```diff
@@ -79,7 +79,27 @@ const Editor = (props: EditorProps) => {
 
   return (
     <LexicalComposer initialConfig={settings}>
-      <Box className='editor-container' bg={'text_area.bg_light'} _dark={{ bgColor: 'text_area.bg_dark' }}>
+      <Box
+        className='editor-container'
+        bg={'text_area.bg_light'}
+        _dark={{
+          bgColor: 'text_area.bg_dark',
+          borderColor: '#4A5568 !important',
```

Comment:
If you need to use `!important` is because you're not doing it the right way. Border color is determined by colorSchemes.